src/graph/expression_operators.cpp

```diff
@@ -1,3 +1,8 @@
+/* All or part of this file was contributed by NVIDIA under license:
```

Comment:
I don't mean to diminish NVIDIA's contribution, but would NVIDIA be fine with having removing the "All or" phrasing from this message?

Reply:
And @ykim362 just told me that technically we are also supposed to add a Microsoft notice like that everywhere. So when Intel starts adding theirs we will have a full screen of notices before we get to any code :)

Reply:
https://www.youtube.com/watch?v=mahTGNIk4q4

Reply:
@kpu .so... full screen of contribution notices it is? :)

Reply:
@emjotde I also had reservations about including this notice in some files where I made small changes. I was told that I had to include those notices with that phrasing even where I made one line changes. I will ask again if I can modify the wording at least. 

Sorry about this :(

Reply:
No reason to be sorry, I know how it works.